<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Change_psw</class>
 <widget class="QWidget" name="Change_psw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:rgb(255, 255, 255);
font-size:15px;
font-family:Century Gothic,sans-serif;
background-image: url(:/qss/psblack/ce75199595b95100538ccfd6827834cd.jpg);
}
QFrame{
background:rgba(0,0,0,0.8);
border-radius:15px;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>361</width>
     <height>511</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="stu_id">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;</string>
    </property>
    <property name="text">
     <string>Student ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="old_psw">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;</string>
    </property>
    <property name="text">
     <string>Old Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_psw">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;</string>
    </property>
    <property name="text">
     <string>New Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirm">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;</string>
    </property>
    <property name="text">
     <string>Confirm your Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:red;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>Save the Change</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:red;
border-radius:60px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/qss/psblack/img_227642.png</normaloff>:/qss/psblack/img_227642.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
